src/components/Button/Button.tsx

```diff
@@ -1,24 +1,47 @@
 import React, { useState } from "react";
 
-import { testIds } from "../Button/testIds.js";
+import { testIds } from "./testIds";
 
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faCheck, faTimes } from "@fortawesome/free-solid-svg-icons";
 
 import styles from "./Button.module.css";
 
-export const Button = ({ onClick, withConfirm = false, text }) => {
+export enum ButtonType {
+  submit = "submit",
+  reset = "reset",
+  button = "button",
+}
+
+export type IProps = {
+  onClick: () => void;
+  text: string;
+  withConfirm?: boolean;
+  type?: ButtonType;
+  disabled?: boolean;
+};
+
+export const Button = ({
+  onClick,
+  withConfirm = false,
+  text,
+  type = ButtonType.button,
+  disabled = false,
+}: IProps) => {
   const [showConfirm, setShowConfirm] = useState(false);
 
   return (
     <div className={styles.wrapper} data-testid={testIds.wrapper}>
       <button
+        className={styles.button}
         data-testid={testIds.btn}
```

Comment:
maybe use it as a prop? To make a difference between different buttons around the app?